crates/biome_js_analyze/src/analyzers/style/use_numeric_literals.rs

```diff
@@ -171,25 +193,25 @@ enum Radix {
 impl Radix {
     fn from_f64(v: f64) -> Option<Self> {
         Some(if v == 2.0 {
-            Self::Binary
+            Radix::Binary
         } else if v == 8.0 {
-            Self::Octal
+            Radix::Octal
         } else if v == 16.0 {
-            Self::Hexadecimal
+            Radix::Hexadecimal
```

Comment:
It's more idiomatic to use the `Self`. Also, if you change the name of `Radix`, you wouldn't need to change these too.